File: bootpay/src/main/java/kr/co/bootpay/EventListener.java
package kr.co.bootpay;

import android.support.annotation.Nullable;

public interface EventListener extends CancelListener, ConfirmListener, ErrorListener, DoneListener {
    @Override
    void onError(@Nullable String message);

    @Override
    void onCancel(@Nullable String message);

    @Override
    void onConfirmed(@Nullable String message);

    @Override
    void onDone(@Nullable String message);
}

File: bootpay/src/main/java/kr/co/bootpay/EventListener.kt
package kr.co.bootpay

interface EventListener : CancelListener, ReadyListener, ConfirmListener, ErrorListener, DoneListener, CloseListener {
    override fun onError(message: String?)

    override fun onCancel(message: String?)

    override fun onClose(message: String?)

    override fun onReady(message: String?)

    override fun onConfirm(message: String?)

    override fun onDone(message: String?)
}
